server/src/routes/api/v1/characterInfoRouter.js

```diff
@@ -0,0 +1,42 @@
+import express from "express";
+import objection from "objection";
+import { Stats, Look } from "../../../models/index.js";
+const { ValidationError } = objection;
+
+import cleanUserInput from "../../../services/cleanUserInput.js";
+
+const characterInfoRouter = new express.Router({ mergeParams: true });
+
+characterInfoRouter.post("/", async (req, res) => {
+  const { stats, look } = req.body;
+  const { charId } = req.params;
+  const { charm, cool, sharp, tough, weird } = stats;
+  const { aura, body, clothes, eyes, face } = look;
+
+  try {
+    const newStats = await Stats.query().insertAndFetch({
+      charm,
+      cool,
+      sharp,
+      tough,
+      weird,
+      characterId: charId,
+    });
+    const newLook = await Look.query().insertAndFetch({
+      aura,
+      body,
+      clothes,
+      eyes,
+      face,
+      characterId: charId,
+    });
+    // return res.status(201).json({ newStats });
```

Comment:
we can remove the commented out code.

And you can see that if we kept all of this character data on one table, that would be at least two fewer DB queries. Again, up to you!